streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/interceptor/UploadFileTypeInterceptor.java

```diff
@@ -43,11 +42,10 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
       Map<String, MultipartFile> files = multipartRequest.getFileMap();
       for (String file : files.keySet()) {
         MultipartFile multipartFile = multipartRequest.getFile(file);
-        Utils.notNull(multipartFile);
+        ApiAlertException.throwIfNull(multipartFile, "File to upload can't be null.");
```

Comment:
If using ApiAlertException, In addition show the err message to user, we also meed to tell user the result of the operation, such as success or failure. 
This message must be simple and precise

Reply:
> If using ApiAlertException, In addition show the err message to user, we also meed to tell user the result of the operation, such as success or failure. This message must be simple and precise

e.g:         ApiAlertException.throwIfNull(multipartFile, "File to upload can't be null. upload jar failed");

Reply:
> 

I'm fine to it.